views/editor/bramble.html

```diff
@@ -113,15 +117,26 @@
                 transform: rotateX(-360deg) rotateY(-360deg);
             }
         }
+	
+	a {
```

Comment:
Here also, please use spaces instead of tabs. Also, nest this rule in ``#spinner-container`` like the rest of the rules.